Microsoft/WebTemplateStudio
# add aria-labels to top nav bar links

Changes:
- added aria labels to top nav bar

How to Test with screen reader:
- go to settings -> ease of access -> narrator 
- turn narrator on
- build the extension and run it
- click on the tabs on top nav bar and you should hear "go to ___ page [link]" where [link] is added in by default
